metrics/web/test/common/presentation/navigation/state/navigation_notifier_test.dart

```diff
@@ -167,13 +168,91 @@ void main() {
       },
     );
 
+    test(
+      ".handleLoggedOut() clears pages when a user logs out",
+      () {
+        notifier.handleLoggedIn();
+        notifier.push(DefaultRoutes.dashboard);
+
+        notifier.handleLoggedOut();
+
+        expect(notifier.pages, hasLength(1));
+      },
+    );
+
+    test(
+      ".handleLoggedOut() pushes to the login page when a user logs out",
+      () {
+        notifier.handleLoggedIn();
+        notifier.push(DefaultRoutes.dashboard);
+
+        notifier.handleLoggedOut();
+
+        final currentPage = notifier.pages.last;
+
+        expect(currentPage.name, isLoginPageName);
+      },
+    );
+
+    test(
+      ".handleLoggedOut() does not clear pages when a user logs out from the login page",
```

Comment:
✅ 
```suggestion
      ".handleLoggedOut() does not clear pages when a user is not logged in",
```
It seems like this test checks that the `handleLoggedOut` method does not clear pages when the user is not logged in. Please, apply this suggestion to the test below.